src/olympia/addons/tests/test_serializers.py

```diff
@@ -1034,10 +1034,12 @@ def test_basic(self):
         self.addon = addon_factory()
 
         result = self.serialize()
-        assert set(result.keys()) == set(['id', 'name', 'icon_url', u'url'])
+        assert set(result.keys()) == set(
+            ['id', 'name', 'icon_url', 'type', 'url'])
```

Comment:
it's a nit, and I realise you're just slightly changing the test so not a blocker, but defining a list inside a set is redundant.

Reply:
it's not, you can't initialize a `set` without an iterable, sadly

Reply:
(unless you want an empty one, that is)

Reply:
`{'id', 'name', 'icon_url', 'type', 'url'}`

Reply:
Oh, silly me, always forget that `set` do have a litteral constructor !